static/style.css

```diff
@@ -3,6 +3,13 @@
 }
 
 .uk-background-secondary .uk-navbar-dropdown-nav .uk-nav-header {
+    color: #666 !important;
```

Comment:
That doesn't work for everything because UIKit has a JS portion. That portion often applies styling directly to elements - which is a problem, as you can imagine.

I always try without `!important` first.